proxy/http/HttpSM.h

```diff
@@ -177,6 +177,20 @@ enum HttpPluginTunnel_t {
 class CoreUtils;
 class PluginVCCore;
 
+class PostDataBuffers
+{
+public:
+  MIOBuffer *postdata_copy_buffer            = nullptr;
+  IOBufferReader *postdata_copy_buffer_start = nullptr;
+  IOBufferReader *ua_buffer_reader           = nullptr;
+
+  void clear();
+  void init(IOBufferReader *ua_reader);
+  void copy_partial_post_data();
+
+  ~PostDataBuffers();
+};
```

Comment:
Just an observation. By omitting the default constructor definition here, we are losing the following debug message from HttpTunnel.h
```
266      Debug("http_redirect", "[PostDataBuffers::PostDataBuffers]"); 
```

This may not have been a very useful debug message though.

Reply:
IMO, `postbuf` in HttpSM is not a pointer. It is always allocated with HttpSM.

Reply:
OK, sounds good. Just checking.